contracts/test/upgrade-test.ts

```diff
@@ -121,14 +121,14 @@ describe("Upgrade", async function () {
     const walletAddress = walletOldVg.address;
     const blsSecret = walletOldVg.blsWalletSigner.privateKey;
 
-    const wallet = await BlsWalletWrapper.connect(
-      blsSecret,
-      fx.verificationGateway.address,
-      fx.verificationGateway.provider,
-    );
     // Sign simple address message
     const addressMessage = solidityPack(["address"], [walletAddress]);
-    const addressSignature = wallet.signMessage(addressMessage);
+    const walletNewVg = await BlsWalletWrapper.connect(
+      blsSecret,
+      vg2.address,
+      vg2.provider,
+    );
+    const addressSignature = walletNewVg.signMessage(addressMessage);
```

Comment:
Now that the message needs to be signed with the VG address, I needed to sign this message with a wallet for the new VG rather than the original wallet